inference-engine/include/cldnn/cldnn_config.hpp

```diff
@@ -159,6 +104,7 @@ DECLARE_CONFIG_KEY(DUMP_KERNELS);
  *
  * For values TUNING_CREATE and TUNING_RETUNE the file will be created if it does not exist.
  */
+INFERENCE_ENGINE_DEPRECATED("The config key will be removed")
```

Comment:
should not we deprecated the keys below as well?

Reply:
Fixed